content/installation/linux_package_repo/ContrastRepoConfig.md

```diff
@@ -0,0 +1,49 @@
+Some of Contrast's products are distributed via Linux packages. This makes installing and staying up to date very easy and familiar as it utilizes the linux packaging and distribution system. In order to use our package repo you have to configure your package management system to include a reference to our package repo.
+
+Some of our packages are tied to the specific Linux distribution you are using and some are applicable to any distribution. This document will configure both distribution points so once configured, you will have access to all our Linux-packaged products.
+
+This is a one-time configuration step. Once its done, you'll be connected to the software distribution pipeline and can solely manage it via your distribution's package manager. Some Linux package managers need a public GPG key that will be used to verify the signed packages from our repo.  We'll show this in the configuration steps if its applicable to your system
+
+### Debian/Ubuntu
+
+Debian-based distributions are identified by their codename.  Codenames for Ubuntu versions are:
+
+	* Ubuntu 18.04: bionic
+	* Ubuntu 16.04: xenial
+	* Ubuntu 14.04: trusty
+
+* First install the public key we use to sign our packages.
+
+```
+curl https://contrastsecurity.jfrog.io/contrastsecurity/api/gpg/key/public | sudo apt-key add -
+```
+
+* Next, add the package repos. Be sure to replace the `CODENAME` below with your distros actual codename. This can be found in ```/etc/os-release```. Create the file ```/apt/sources.list.d/contrast.list``` with your favorite editor and add the content below to it:
```

Comment:
Copy edit: we use the shorthand "repo" instead of "repository" a lot but not always. We should standardize on one, and I think we should favor the long form instead of shorthand

Reply:
changed mentions of it in this doc to the long form.